notebooks/yolo-detection-classification/yolo-detection-classification.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<p>Don't add please image in the repo. You can add the images to any comment in PR and use that link for downloading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>